Source/WTF/wtf/PlatformEnableCocoa.h

```diff
@@ -131,7 +131,7 @@
 
 #endif // !PLATFORM(MAC) && !PLATFORM(MACCATALYST) && !PLATFORM(WATCHOS) && !PLATFORM(APPLETV)
 
-#if !defined(ENABLE_APPLE_PAY_SESSION_V11) && ((PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 110300) || (PLATFORM(IOS) && __IPHONE_OS_VERSION_MIN_REQUIRED >= 140500))
+#if !defined(ENABLE_APPLE_PAY_SESSION_V11) && ((PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 110300) || PLATFORM(IOS))
```

Comment:
__MAC_OS_X_VERSION_MIN_REQUIRED >= 110300 is always true, we don't support targeting 11.0 at this point.